osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -378,6 +388,9 @@ protected override void OnMouseUp(MouseUpEvent e)
 
         protected override bool OnClick(ClickEvent e)
         {
+            //triangles.AccentColours = this.FindClosestParent<MainMenu>()!.GetBackgroundColours(2).Select(x => Graphics.Backgrounds.Triangles.InRange(x, 0)).ToArray();
+            //background.Colour = this.FindClosestParent<MainMenu>()!.GetBackgroundColours()[0];
+
```

Comment:
Why is this removed?

Reply:
Oh, my bad, parts left from old PR with multicolored triangles.